src/wp-admin/includes/plugin.php

```diff
@@ -1414,6 +1414,51 @@ function add_comments_page( $page_title, $menu_title, $capability, $menu_slug, $
 	return add_submenu_page( 'edit-comments.php', $page_title, $menu_title, $capability, $menu_slug, $function );
 }
 
+/**
+ * Add submenu page to the Security main menu.
+ *
+ * The function which is hooked in to handle the output of the page must check
+ * that the user has at least 'manage_options' capability.
+ *
+ * @param string   $page_title The text to be displayed in the title tags of the page when the menu is selected.
+ * @param string   $menu_title The text to be used for the menu.
+ * @param string   $menu_slug  The slug name to refer to this menu by; must match an active plugin slug.
+ * @param callable $function   The function to be called to output the content for this page.
+ * @return false|string The resulting page's hook_suffix, or false if the user does not have the 'manage_options' capability.
+ */
+function add_security_page( $page_title, $menu_title, $menu_slug, $function = '' ) {
+	$mu_plugins = get_mu_plugins();
+	$plugins    = array_merge(
+		(array) get_option( 'active_sitewide_plugins' ),
+		(array) get_option( 'active_plugins' ),
```

Comment:
Let's add `, array()` here to set a default, this pattern is used in other parts of the code.